editor/store/defaults.js

```diff
@@ -15,46 +20,57 @@ export const EDITOR_SETTINGS_DEFAULTS = {
 	alignWide: false,
 	colors: [
 		{
-			name: 'pale pink',
+			name: __( 'pale pink' ),
+			slug: 'pale pink',
```

Comment:
Are these supposed to have dashes?

Reply:
they probably should, but they get sanitized a bit further down the road. 

So, yes for understandability, no for actual need.